### ISSUE TITLE
Onfido 25.0.0 Custom Fonts would not apply on all screens

### ISSUE THREAD

**Issue by Zeropointer11** [onfido/onfido-ios-sdk#292]
### What was the expected behaviour?

When you set a Custom Font at the Appearance() it should be used on all screens.

### What happened instead?

some Screens did have the default Font and only the Buttons have the custom Font set

![IMG_5E708AE714D1-1](https://user-images.githubusercontent.com/3930333/170001175-1b91f86f-a47d-45f4-9207-8ab9fa3a2b63.jpeg)

![IMG_6634E54A60BF-1](https://user-images.githubusercontent.com/3930333/170001211-38ae8856-c266-42be-9429-db51d04701a9.jpeg)

### Version info:

- Cocoapods (if applicable):
- Onfido SDK (Debug/Release): Debug 
- iOS: 15.5
- Xcode: 13.4
- Device/Simulator: iPhone SE2 & iPhone 11 Pro
- Device/Simulator language: de

### Integration configuration:

```
 let appearance = Onfido.Appearance()
            appearance.primaryColor = UIColor(Color.red)
            appearance.primaryTitleColor = UIColor(Color.white)
            appearance.primaryBackgroundPressedColor = UIColor(Color.red.opacity(0.5))
            appearance.secondaryBackgroundPressedColor = UIColor(Color.red.opacity(0.3))
            appearance.fontRegular = "Chalkduster"
            appearance.fontBold = "MarkerFelt-Wide"

let config = try OnfidoConfig.builder()
                .withSDKToken(token)
                .withWelcomeStep()
                .withDocumentStep(ofType: .drivingLicence(config: nil))
                .withFaceStep(ofVariant: .photo(withConfiguration: .none))
                .withAppearance(appearance)
                .withCustomLocalization(andTableName: "OnfidoLocalizable_DE", in: .main)
                .build()

let onfidoFlow = OnfidoFlow(withConfiguration: config)
                .with(responseHandler: { [weak self] response in 
//TODO: add response handler
},
                      dismissFlowOnCompletion: true)

 let viewController = try onfidoFlow.run()
 navigationController.present(viewController, animated: true)

```

### Steps to reproduce:

add Onfido to your Project `.package(url: "https://github.com/onfido/onfido-ios-sdk.git", from: "25.0.0")`
Init an Onfido.Apperance() set it in the OnfidoConfig.builder() and set the Config to OnfidoFlow(withConfiguration: config).run()

**Comment by kushal-onfido** [comment]
Hello @Zeropointer11 

This issue has been fixed in the latest SDK release [25.2.0](https://github.com/onfido/onfido-ios-sdk/releases/tag/25.2.0). Could you please upgrade the SDK to the latest version and confirm if everything is good now?

Thank you!

**Comment by Zeropointer11** [comment]
Fixed with version 27.3.0 sry for the late replay
